runtime/oti/j2sever.h

```diff
@@ -112,5 +114,10 @@
  */ 
 #define J2SE_SHAPE_FROM_ENV(env) J2SE_SHAPE(((J9VMThread*)env)->javaVM)
 
+/**
+ * Macro to extract java spec version from J2SE version.
+ */
+#define JAVA_SPEC_VERSION_FROM_J2SE(j2seVersion) 	(j2seVersion>> J2SE_JAVA_SPEC_VERSION_SHIFT)
```

Comment:
Missed it before, j2seVersion should be (j2seVersion) when used in the macro, with a space afterward

Reply:
Fixed